clwb/src/com/google/idea/blaze/clwb/run/BlazeCidrLauncher.java

```diff
@@ -215,7 +215,8 @@ public CidrDebugProcess createDebugProcess(CommandLineState state, XDebugSession
         workingDir = workspaceRootDirectory;
```

Comment:
This was properly set above as `<target>.runfiles/<workspace_name>` (with a fallback to workspace root dir) but never used past this line.